python/pyarrow/tests/parquet/conftest.py

```diff
@@ -81,6 +81,7 @@ def s3_example_fs(s3_server):
     host, port, access_key, secret_key = s3_server['connection']
     uri = (
         "s3://{}:{}@mybucket/data.parquet?scheme=http&endpoint_override={}:{}"
+        "&allow_bucket_creation=True"
```

Comment:
Do you remember why this was needed?

Reply:
```python
    @pytest.fixture
    def s3_example_fs(s3_server):
        from pyarrow.fs import FileSystem
    
        host, port, access_key, secret_key = s3_server['connection']
        uri = (
            "s3://{}:{}@mybucket/data.parquet?scheme=http&endpoint_override={}:{}"
            .format(access_key, secret_key, host, port)
        )
        fs, path = FileSystem.from_uri(uri)
    
>       fs.create_dir("mybucket")

pyarrow/tests/parquet/conftest.py:88: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
pyarrow/_fs.pyx:603: in pyarrow._fs.FileSystem.create_dir
    check_status(self.fs.CreateDir(directory, recursive=recursive))
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

>   raise convert_status(status)
E   OSError: Bucket 'mybucket' not found. To create buckets, enable the allow_bucket_creation option.
```

Reply:
Looks like there is something off with my setup. I have removed the change and the tests are not failing on the CI.